src/Process/Runner/MutationTestingRunner.php

```diff
@@ -100,6 +103,17 @@ public function run(iterable $mutations, string $testFrameworkExtraOptions): voi
 
                 return false;
             })
+            ->filter(function (Mutant $mutant) {
+                if ($mutant->getMutation()->getNominalTestExecutionTime() < $this->timeout) {
```

Comment:
@danepowell if we add return false above this line we should be able to see that we are getting significantly better results if your issue https://github.com/infection/infection/issues/1965 is indeed a bug